### DISCUSSION TITLE
"Create Redact Annotation" command

### DISCUSSION THREAD

**Discussion by Tropinin** [sumatrapdfreader/sumatrapdf#4505] [Q&A]
Hi! i'm trying to add a shortcut for the command "Create Redact Annotation", but it doesn't seem to work. I tried replacing it with other commands from the docs, and those work. Is "Create Redact Annotation" not implemented or I'm doing something wrong?
I'm using the newest version of Sumatra
There's the shortcuts part of the settings:
```
Shortcuts [
	[
		Cmd = CmdCreateAnnotRedact
		Key = Shift + A
	]
]
```

**Comment by kjk** [comment]
This one is not implemented yet. Please open an issue to track this.
